modules/accessories.js

```
@@ -0,0 +1,32 @@
+const form = document.querySelector('.add-new');
+const contactSection = document.querySelector('.contact-section');
+const bookSection = document.querySelector('.book-list');
+
+const initialise = () => {
+  form.classList.add('hide');
+  contactSection.classList.add('hide');
+};
```

- [x] [_OPTIONAL_] Great job, for a few seconds I can see all the pages at once in the browser, one of the reasons might be the time it takes to run the javascript, I think a better approach could be by default giving them the class hide and when the link is clicked removing it.

---

Thank you @Wes-Isaac that's a good suggestion. Let me implement it 👍